backend/src/main/java/pl/edu/agh/awiteks_backend/services/PlantService.java

```diff
@@ -59,4 +65,25 @@ private void removePlantFromUserList(int id) {
                         .removePlant(presentPlant)
         );
     }
+
+    private List<Plant> getUsersPlants(int userId) {
+        // TODO maybe create custom exception
+        return userRepository.get(userId)
+                .map(User::getUserPlants)
+                .orElseThrow(() -> new IllegalArgumentException("no such user"));
+    }
+
+    public List<PlantSummary> getPlantSummaries(int userId) {
+        return getUsersPlants(userId).stream()
+                .map(PlantMapper::plantToPlantSummary)
+                .collect(Collectors.toList());
+    }
+
+    public PlantsStats getPlantsStats(int userId) {
+        // TODO move frontend logic to compute these stats here
+        // TODO we can either aggregate this or compute every time
```

Comment:
Dodałbym to również jako taski na Jirze, jeśli ich nie ma

Reply:
Jak chcesz to dodaj, nie wiem czy chcemy PR'a na pare linijek robic czy przy okazji to zrobicie

Reply:
Można wsm przy okazji jakiegoś taska to machnąć